docs/md/BasicTutorial.md

```diff
@@ -0,0 +1,282 @@
+# Tutorial
+
+
+
+## Installation
+Install the python client and check the installation as follows:
+
+```command
+$ pip install osparc
+$ python -c "import osparc; print(osparc.__version__)"
+```
+
+
+## Setup
+
+To setup the client, we need to provide a username and password to the configuration. These can be obtained in the UI under [Preferences > API Settings > API Keys](http://docs.osparc.io/#/docs/platform_introduction/main_window_and_navigation/user_setup/profile?id=preferences). Use the *API key* as username and the *API secret* as password. For security reasons, you should not write these values in your script but instead set them up via environment variables or read them from a separate file. In this example, we use environment variables
```

Comment:
just add "which will be referred to as "OSPARC_API_KEY" and "OSPARC_API_SECRET" for the rest of the tutorial."